### PR TITLE
Set and document ttls

### PR DESCRIPTION
**Pull request by rudigiesler**
For storing event URLs, the default should be 2 days, for replies, the default should be 10 minutes. These defaults and their implications should be documented, and the defaults should be set correctly.

### CODE REVIEW

**Comment by rudigiesler** [comment]
Originally I also wanted to update the optional arguments for running `jb`, but it turns out this is complex due do sphinx not wanting to have multiple character options with a single `-`, i.e. sphinx breaks when you have `-redish`. This will be done in another PR when we've found a solution.

File: docs/http_api.rst [hunk 1]
```diff
@@ -132,9 +132,13 @@ Messages
        the uuid of the message being replied to if this is a response to a
        previous message. Important for session-based transports like USSD.
        Optional. Only one of ``to`` or ``reply_to`` may be specified.
+       Defaults settings allow 10 minutes to reply to a message, after which
```
Comment (hodgestar) [inline docs/http_api.rst]: s/Defaults/Default/ or maybe `The default ...`?

File: junebug/command_line.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def parse_arguments(args):
     parser.add_argument(
         '--inbound-message-ttl', '-ittl', dest='inbound_message_ttl', type=int,
         help='The maximum time allowed to reply to a message. Defaults to '
-        '"60" seconds')
+        '60*10 seconds or 10 minutes')
```
Comment (hodgestar) [inline junebug/command_line.py]: I think we need to reword this to say something like `The maximum time allowed to reply to a message (in seconds). Defaults to 600 seconds (10 minutes)`, otherwise it becomes unclear what the units are.

File: docs/http_api.rst [hunk 3]
```diff
@@ -132,9 +132,13 @@ Messages
        the uuid of the message being replied to if this is a response to a
        previous message. Important for session-based transports like USSD.
        Optional. Only one of ``to`` or ``reply_to`` may be specified.
+       The default settings allow 10 minutes to reply to a message, after which
+       an error will be returned.
    :param str event_url:
        URL to call for status events (e.g. acknowledgements and
-       delivery reports) related to this message.
+       delivery reports) related to this message. Default settings allow for
```
Comment (hodgestar) [inline docs/http_api.rst]: Do we want to change this to `The default settings ...` too? And the one further down?

**Comment by hodgestar** [comment]
Very minor comments, otherwise looks good.

**Comment by rudigiesler** [comment]
Comments should be addressed.

**Comment by hodgestar** [comment]
:+1:
